site/src/pages/components/combobox.explainer.mdx

```diff
@@ -0,0 +1,277 @@
+
+Further highlighting the need for such a solution, a recent [Twitter poll](https://twitter.com/openuicg/status/1717345237889351911) run by the Open UI Community Group underscored the widespread demand for a standardized ``combobox``
+<Image
+  src="/images/combobox/combobox-twitterpoll.png"
+  alt="Combobox twitter poll results"
+/>
+
+
+This proposal for `<combobox>` builds upon the strengths of `<selectlist>` and `<listbox>`. It aims to offer a control that is versatile for developers, intuitive for users, and, above all, accessible to all.
+
+By standardizing such elements, we hope to bridge the gap between design demands and the need for performant, reliable, and accessible web controls. 
+
+### Scope of this document
+
+This document aims to provide a complete view of combo-box utilisation across the web to inform a potential proposal within Open UI. The goal of this document is to propose a very basic anatomy to simplify the initial shipment of `<combobox>` (or some other name) in the web platform. It will heavily build upon `selectlist` but provide a better accessible and extensible solution than that of an `input` bound with `datalist`.
+
+
+### Out of scope of this document
+
+This document will not be attempting some of the complex scenarios found across the web that leverage various ways in which to represent multi-selected options. Additionally, it does not intend to cover the addition of options that are not available in a `<listbox>`. It will not cover multi-select but will leverage the resolutions defined by `selectlist`.
+
+A good analogy for what this document will result in looking like is the [Tabs research document.](https://open-ui.org/components/tabs.research.parts/)
+
+## Recommended anatomy for version 1
+
+```
+<combobox>
+   <input type=selectlist>
+   <listbox>
+	<option>One</option>
+	<option>Two</option>
+   </listbox>
+</combobox>
+```
+
+
+**Pros:**
+* Clear Semantics: The `<combobox>` element explicitly conveys its purpose.
+
+**Cons:**
+* Introducing new element, `<combobox>`
+
+### Anatomy of `<combobox>`:
+
+<ComboboxAnatomy />
+
+- `<combobox>`: The root container that encapsulates the entire `combobox` structure. It provides context for the interaction between the `input` and the `listbox`.
+  - Slots:
+    - `<input>`
+    - `<listbox>`
+  - Attributes:
+    - **search**: Indicates the combobox will actively fetch results based on user input.
+    - **filter**: Indicates the combobox will narrow down visible options based on user input.
+    - **multiple**: Allows multiple options to be selected.
+- `<input type=selectlist>`: This is where the user can type input or view the selected option. It acts as the trigger to display the listbox.
+  - Slots:
+    - **placeholder** (optional): To display hint text when the input is empty.
+    - **selectedoption** (optional): To display the currently selected option from the `listbox`. 
+      - Note: Currently input doesn't support slots
```

Comment:
Based on the above two comments, it does due to it being a part of the user agent's shadow tree for an `input`. Follow what the `selectlist` spec does here and show these slots. We will not be adjusting how `input` works. I do think that `combobox` needs to support the `selectedoption` element but I'm unsure how to present it in the anatomy itself without it seeming like we're replacing `value` which we don't want to do. Thoughts @mfreed7 @josepharhar ?